api/renterd/client.go

```diff
@@ -124,7 +124,7 @@ func (c *Client) RHPAppend(rar RHPAppendRequest, sector *[rhp.SectorSize]byte) (
 }
 
 // HostDBHosts gets a list of hosts in the host DB.
-func (c *Client) HostDBHosts() (resp []hostdb.Host, err error) {
+func (c *Client) HostDBHosts() (resp []types.PublicKey, err error) {
```

Comment:
I think this should return a paginated list of `hostdb.Host`. Returning just the public keys means consumers will need to make one additional request per host for useful data. It's not too bad, but much less convenient.